sql/internal/specutil/types.go

```diff
@@ -83,6 +86,17 @@ func (r *TypeRegistry) Register(specs ...*schemaspec.TypeSpec) error {
 	return nil
 }
 
+func validSpec(typeSpec *schemaspec.TypeSpec) error {
+	var seenOptional bool
```

Comment:
```suggestion
	var optionalSeen bool
```

you may know what is more correct in terms of grammar.

Reply:
i think both are ok. 
i meant it as "I've already seen the optional" and the way you put it is "an optional param was seen" 
leaving as is